assets/base.css

```diff
@@ -2197,14 +2205,10 @@ product-info .loading-overlay:not(.hidden) ~ *,
   box-shadow: 0 0 0 -0.2rem rgb(var(--color-foreground));
 }
 
-@media screen and (min-width: 750px) {
-  .announcement-bar .slider-button--next {
-    margin-right: -3.2rem;
-  }
-
-  .announcement-bar .slider-button--prev {
-    margin-left: -3.2rem;
-  }
+.utility-bar__grid--3-col .desktop-localization-wrapper,
+.utility-bar__grid--2-col .desktop-localization-wrapper {
+  grid-area: language-currency;
+  align-self: center;
 }
```

Comment:
I believe the `.desktop-localization-wrapper` is used only in this context (for the locales in the header on desktop) so we could probably just write: 
```css
.desktop-localization-wrapper {
  grid-area: language-currency;
  align-self: center;
}
```